pkg/auth/credentials.go

```diff
@@ -117,6 +117,11 @@ func (cred Credentials) IsExpired() bool {
 	return cred.Expiration.Before(time.Now().UTC())
 }
 
+// IsTemp - returns whether credential is temporary or not.
+func (cred Credentials) IsTemp() bool {
+	return cred.SessionToken != "" && !cred.IsExpired()
```

Comment:
`IsExpired` does not seem relevant here. We should remove it.

Reply:
It is relevant @donatello if you are adding a user with same name and it is already expired we should allow them to add

Reply:
There seems to be `IsValid` for this purpose.

Reply:
Yeah but it won't tell you if its a temporary token @donatello

Reply:
I guess we don't need to worry about expired creds, perhaps they will be flushed away anyways.